pkg/neotest/compile.go

```diff
@@ -90,3 +92,30 @@ func CompileFile(t testing.TB, sender util.Uint160, srcPath string, configPath s
 	contracts[cacheKey] = c
 	return c
 }
+
+// LoadContract loads a contract from the specified NEF and manifest files.
+func LoadContract(t testing.TB, sender util.Uint160, nefPath, manifestPath string) (*Contract, error) {
```

Comment:
`LoadContract` associates with VM and context loading whereas this function is not related to VM. I'd say that it's more like `ReadContract` or `ReadNEF`.